FlowCrypt/src/main/java/com/flowcrypt/email/jetpack/workmanager/HandlePasswordProtectedMsgWorker.kt

```diff
@@ -135,11 +137,18 @@ class HandlePasswordProtectedMsgWorker(context: Context, params: WorkerParameter
 
             //start of creating and uploading a password-protected msg to FES
             val fromAddress = plainMimeMsgWithAttachments.getFromAddress()
-            val domain = EmailUtil.getDomain(fromAddress.address)
+            val baseFesUrlPath = if (account.useFES) {
```

Comment:
`account.useFES` is ambiguous, both options are FES now. One of them is "custom url fes" and the other one is "shared tenant fes". So it should be called `account.useCustomUrlFes`